### DISCUSSION TITLE
Import Template - Axio

### DISCUSSION THREAD

**Discussion by joelabc** [ananthakumaran/paisa#176] [General]
Hey @ananthakumaran I left a comment on the Reddit thread regarding the issue I faced with importing a specific format. Here is the anonymized version of the report from Axio [https://axio.co.in/]
[Axio - Anon Export.csv](https://github.com/ananthakumaran/paisa/files/14235009/Axio.-.Anon.Export.csv)

**Comment by ananthakumaran** [comment]
```handlebars
{{#if (isDate ROW.A "YYYY-MM-DD")}}
  {{date ROW.A "YYYY-MM-DD"}} {{ROW.C}}
  {{#if (eq ROW.E "DR")}}
    {{or (match ROW.I
          Expenses:Shopping="SHOPPING" 
          Expenses:Groceries="FOOD & DRINKS")
         "Expenses:Unknown"}}     {{amount ROW.D}} INR
    Assets:Checking
  {{else}}
    Assets:Checking                          {{amount ROW.D}} INR
    {{predictAccount ROW.I prefix="Income"}}
  {{/if}}
{{/if}}
```

You can try the above template, since this includes many types of transactions, the account prediction would likely be bad and you might have to fix it by hand. You can tweak the expense categories etc in the template (just add more lines in the same format). The date and amount part should work without any issue though.
